webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala

```diff
@@ -137,11 +135,11 @@ class VolumeTracingService @Inject()(
                   ))
               case a: RevertToVersionVolumeAction =>
                 revertToVolumeVersion(tracingId, a.sourceVersion, updateGroup.version, tracing)
-              case a: DeleteSegmentDataVolumeAction =>
+              /*case a: DeleteSegmentDataVolumeAction =>
                 if (!tracing.getHasSegmentIndex) {
                   Fox.failure("Cannot delete segment data for annotations without segment index.")
                 } else
-                  deleteSegmentData(tracingId, tracing, a, segmentIndexBuffer, updateGroup.version) ?~> "Failed to delete segment data."
+                  deleteSegmentData(tracingId, tracing, a, segmentIndexBuffer, updateGroup.version) ?~> "Failed to delete segment data."*/
```

Comment:
How difficult would it be to implement this feature for ND? If it’s not straightforward, I’d vote to add an assertion here, and also disable it in the frontend for ND datasets. Commenting it out is probably not the solution :D

Reply:
Don't know why I commented this originally, but I uncommented it now. @dieknolle3333 I think the delete segment data action currently does not provide additional coordinates.

Reply:
Do you want me to disable this context-menu-action in the frontend? Or adjust the action that it provides additional coords? Do we all agree that this should delete the segments data in the current additional coord tuple (and not everywhere)?
I looked into it shortly and I think it would be straight-forward to implement, though I would prefer to do it in a quick and separate PR and disable it for now.
@philippotto what's your opinion on this?

Reply:
> Do we all agree that this should delete the segments data in the current additional coord tuple (and not everywhere)?

Hm, I think it would make more sense to delete it everywhere (it's not called "delete_segment_in_current_additional_axes"). Can the back-end do this in one go (i.e., not requiring one request per coord tuple)? If so, I'd prefer this.

**If not**, the following sounds like a good plan. The user should be warned then that the deletion is only done in the current coords.

> I looked into it shortly and I think it would be straight-forward to implement, though I would prefer to do it in a quick and separate PR and disable it for now.